src/datepicker/ngb-date.ts

```diff
@@ -0,0 +1,25 @@
+export class NgbDate {
+  static from(date: {year: number, month: number, date?: number}) {
```

Comment:
This PR should stick to minimal features as possible, that would be a feature request for the future.